common/src/main/kotlin/uk/gov/justice/digital/hmpps/prisonersearch/common/model/translator.kt

```diff
@@ -166,6 +159,21 @@ fun IncentiveLevel?.toCurrentIncentive(): CurrentIncentive? = this?.let {
   )
 }
 
+fun Prisoner.setRestrictedPatient(
+  rp: RestrictedPatient?,
+  ob: OffenderBooking,
+) {
+  this.restrictedPatient = rp != null
+  this.locationDescription = rp
+    ?.let { "${ob.locationDescription} - discharged to ${it.dischargedHospital?.description}" }
+    ?: ob.locationDescription
```

Comment:
this is a bit odd - if they are not a restricted patient then this is still set even though the method is called `setRestrictedPatient`